msm/src/constraint.rs

```diff
@@ -1,3 +1,7 @@
+use ark_ff::Zero;
```

Comment:
Let's move/clean this in the subdirectory in the next PR

Reply:
Moved in https://github.com/o1-labs/proof-systems/pull/1904